examples/standalone_flow_matching.ipynb

```diff
@@ -27,14 +27,19 @@
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, dim))\n",
-    "    \n",
+    "\n",
+    "    # Batch-aware forward call\n",
     "    def forward(self, t: Tensor, x_t: Tensor) -> Tensor:\n",
     "        return self.net(torch.cat((t, x_t), -1))\n",
```

Comment:
```suggestion
    "    def forward(self, t: Tensor, x_t: Tensor) -> Tensor:\n",
    "        """Network inputs: time t (shape [batch_size, 1]) and x_t (shape [batch_size, dim])."""\n",
    "        return self.net(torch.cat((t, x_t), -1))\n",
```
Lets move this comment into the function as a docstring and clarify the input shapes.